content/blog/20230829_schedule-for-td-agent-4-eol.md

```diff
@@ -0,0 +1,33 @@
+# Drop schedule announcement about EOL of Treasure Agent (td-agent) 4
+
+Hi users and developers!
+
+We announce the dropping schedule for TD Agent 4 development.
+
+## About Treasure Agent (td-agent) 4
+
+Since td-agent 4.0.0 was released in 2020, td-agent 4 had been maintained for a long time.
```

Comment:
I'm not sure, but I think `has` would be better because it is still maintained at this time.

```suggestion
Since td-agent 4.0.0 was released in 2020, td-agent 4 has been maintained for a long time.
```